projects/planner/src/app/gas-blender/gas-blender.component.spec.ts

```diff
@@ -50,4 +50,15 @@ describe('GasBlenderComponent', () => {
 
         expect(calculateSpy).toHaveBeenCalledTimes(1);
     });
+
+    it('Toggle pricing', () => {
+        component.showPricing = true;
```

Comment:
design gap, the showPricing purpose is not to use it to assign the value.
Its done using toglePricing.